libexec/cli/instance.list.exec

```diff
@@ -33,6 +33,67 @@ if ! USERID=`id -ru`; then
     exit 255
 fi
 
+jsonFmt (){
+  INSTANCE='"instance"'
+  INSTANCE_PID='"pid"'
+  INSTANCE_IMG='"img"'
+  JSON=""
+  JSON_INIT='{"instances":['
+  JSON_END=']}'
+  COUNT=0
+
+
+  if [ -z "${1:-}" ]; then
+      DAEMONS=`singularity_daemon_glob '*' | sort | uniq`
+  else
+      DAEMONS=`singularity_daemon_glob "$@" | sort | uniq`
+  fi
+
+DLEN=0
+for i in $DAEMONS; do
+	let "DLEN++"
+done
+let "DLEN--"
+
+  for i in $DAEMONS; do
+      if [ -f "$i" ]; then
+          if [ ! -s "$i" ]; then
+              rm "$i"
+              continue
+          fi
+          FILE_NAME=`basename "$i"`
+          DAEMON_NAME=${FILE_NAME##*-}
+          . "$i"
+          if [ -n "${DAEMON_PID:-}" ]; then
+              PROC_CMDLINE="/proc/${DAEMON_PID:-}/cmdline"
+              if [ -e "$PROC_CMDLINE" ]; then
+                  PROGNAME=$(<"${PROC_CMDLINE}" tr -d \\0)
+                  if [ "${PROGNAME}" != "singularity-instance: $USER [$DAEMON_NAME]" ]; then
+                      message WARN "Removing stale daemon file: $i\n"
+                      rm -f "$i"
+                      continue
+                  fi
+              else
+                  rm -f "$i"
+                  continue
+              fi
+          fi
+
```

Comment:
All of this code is basically just copied and pasted from the code below with the one change of constructing a json instead of printing each line.  That is not going to be very sustainable if we have to change this algorithm.  Can you just wrap the if statement below in another if statement so that it prints if the json environmental variable is not set and it creates the json if the json env var is set?